python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py

```diff
@@ -386,37 +467,39 @@ class OpRuntimeMetrics(metaclass=OpRuntimesMetricsMeta):
         description="Time spent in task output backpressure.",
         metrics_group=MetricsGroup.TASKS,
     )
-    histogram_buckets_s = [
-        0.1,
-        0.25,
-        0.5,
-        1.0,
-        2.5,
-        5.0,
-        7.5,
-        10.0,
-        15.0,
-        20.0,
-        25.0,
-        50.0,
-        75.0,
-        100.0,
-        150.0,
-        500.0,
-        1000.0,
-        2500.0,
-        5000.0,
-    ]
-    task_completion_time: float = metric_field(
```

Comment:
Let's keep existing metrics -- these are metrics used internally (in state telemetry dumped into logs, in the new resource allocator)